velox/common/memory/ByteStream.h

```diff
@@ -383,6 +383,14 @@ inline Timestamp ByteStream::read<Timestamp>() {
   return value;
 }
 
+template <>
+inline LongDecimal ByteStream::read<LongDecimal>() {
+  auto upper = read<uint64_t>();
+  auto lower = read<uint64_t>();
+  LongDecimal value(upper, lower);
```

Comment:
Can we do what Timestamp does?
``readBytes(reinterpret_cast<uint8_t*>(&value), sizeof(value));``